### DISCUSSION TITLE
4K60帧/Hires无损音质

### DISCUSSION THREAD

**Discussion by AYSE1999** [share121/inter-knot#571] [其他]
### ### ![72048990_p0](https://github.com/user-attachments/assets/f3eb2f1e-fe35-45e1-8718-263cb2b36f3e)
![40487102_p0](https://github.com/user-attachments/assets/0009e07b-58a0-4969-a31a-6c98d4c83fab)

![72048990_p0 (1)](https://github.com/user-attachments/assets/b853ac72-41c2-43b2-8ac3-3bee45456fec)
![80680801_p0 (3)](https://github
![90487779_p0 (2)](https://github.com/user-attachments/assets/35a74354-1bde-43a7-92f9-900f55d1d7d4)
.com/user-attachments/assets/f6856213-3df3-4e78-9500-91b0e555ac3b)

**Comment by AYSE1999** [comment]
1

**Comment by AYSE1999** [comment]
图文显示还有问题

**Comment by wjsksk** [comment]
2
